oak_runtime/src/node/wasm/mod.rs

```diff
@@ -815,15 +815,9 @@ impl WasmNode {
             ConfigurationError::IncorrectWebAssemblyModuleName
         })?;
 
-        let signatures = signature_table
-            .values
-            .get(&node_configuration.wasm_module_name)
-            .cloned()
-            .unwrap_or_default();
-
         // We compute the node privilege once and for all at start and just store it, since it does
         // not change throughout the node execution.
-        let node_privilege = wasm_node_privilege(&wasm_module_bytes, signatures.as_ref());
+        let node_privilege = wasm_node_privilege(&wasm_module_bytes, signature_table);
```

Comment:
I'm curious why this was changed, did the previous one not work any more?

Reply:
Yes, it was a bug